saleor/checkout/complete_checkout.py

```diff
@@ -1163,6 +1205,22 @@ def _create_order_from_checkout(
         prices_entered_with_tax=prices_entered_with_tax,
     )
 
+    # update undiscounted order total
+    undiscounted_total = (
+        sum(
+            [line.line.undiscounted_total_price for line in order_lines_info],
+            start=zero_taxed_money(taxed_total.currency),
```

Comment:
I'm wondering why we're changing this. Maybe it's correct, but I'm just checking if it won't cause unexpected calculation changes.

Reply:
@maarcingebala I've found out, that we fixed checkoutComplete mutation, but we didn't change the part related to the orderCreateFromCheckout. So I copy&paste the fix from the checkoutComplete.